larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -358,9 +366,9 @@ TfLiteStatus Prepare(KernelType kernel_type, const int bitwidth,
   return kTfLiteOk;
 }
 
-template <KernelType kernel_type, int bitwidth>
+template <KernelType kernel_type, const int default_bitpacking_bitwidth>
```

Comment:
`const` is not required here. the template arguments are passed at compile-time.